lib/topics/index.js

```diff
@@ -29,7 +29,7 @@ Topics.prototype.getTopic = function(topicName, prefix) {
 
 function wrapInMediatorPromise(self, method, fn) {
```

Comment:
Definitely going to do this as part of a new issue+PR, along with the bug fixes :slightly_smiling_face: